templates/projects/_helpers.html

```diff
@@ -319,7 +319,7 @@ <h2><strong> {{_('Overall progress')}}</strong>: {{ overall_progress | round | i
 {% set is_admin_or_subadmin_or_owner = current_user.admin or current_user.subadmin or current_user.id in project.owners_ids %}
 {% set enable_browse_task = is_admin_or_subadmin_or_owner or (not current_user.admin and not current_user.subadmin and project.info.allow_taskrun_edit) %}
     {% if project['contrib_button'] == 'completed' %}
-    <a href="{{url_for('project.tasks', short_name=project.short_name)}}" class="btn btn-lg btn-primary btn-contribute pull-right">{{ _('Done! View results') }}</a>
+    <a href="{{url_for('project.tasks', short_name=project.short_name)}}" class="btn btn-lg btn-primary btn-contribute btn-view-results pull-right">{{ _('Done! View results') }}</a>
```

Comment:
I wonder if the class name `btn-view-results` (as opposed to `btn-contribute-btn-view-results`) departs from naming conventions because in other places in the code, we have something like `btn-group btn-contribute btn-contribute-now-unpublished` to signify that `now-unpublished` is in the `btn-contribute` class group.

Reply:
This is a good suggestion, that perhaps we can consider more consistent naming conventions. Thank you.